assets/js/blocks/cart-checkout/checkout-i2/inner-blocks/checkout-sample-block/block.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "woocommerce/checkout-sample-block",
+	"version": "1.0.0",
+	"title": "Checkout Sample Block",
+	"category": "woocommerce",
+	"description": "An example block showing how to integrate with Checkout i2.",
+	"supports": {
+		"html": false,
+		"align": true,
+		"multiple": true,
+		"reusable": false
+	},
+	"attributes": {
+		"lock": {
+			"type": "object",
+			"default": {
+				"remove": true
+			}
+		}
+	},
+	"parent": [
+		"woocommerce/checkout-shipping-address-block",
+		"woocommerce/checkout-billing-address-block",
+		"woocommerce/checkout-contact-information-block",
+		"woocommerce/checkout-fields-block",
+		"woocommerce/checkout-totals-block"
+	],
+	"textdomain": "woo-gutenberg-products-block",
+	"editorScript": "file:../../../../../../../build/checkout-blocks/sample.js"
```

Comment:
That's some hola hoop jumping here? I think `editorScript` is relevant only for php (when running `register_block_type`) so script it if you're not using that function.

This makes me thing that the current file placing is questionable.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4684/commits/b91156e0e6c400850c5b11c55c2bdb987422a71d